src/site/content/en/blog/persistent-storage/index.md

```diff
@@ -0,0 +1,166 @@
+---
+title: Persistent storage
+subhead: Persistent storage can help protect critical data from eviction, and reduce the chance of data loss.
+authors:
+  - petelepage
+description: Persistent storage can help protect critical data from eviction, and reduce the chance of data loss.
+date: 2020-05-07
+updated: 2020-05-07
+tags:
+  - blog
+  - storage
+  - progressive-web-apps
+hero: hero.jpg
+alt: PCB circuit board of electronic device
+---
+
+When faced with storage pressure like low disk space, browsers will
+typically [evict data][eviction], including from the Cache API and IndexedDB,
+from the least recently used origin. This may cause data loss if the app
+hasn't synced data to the server, and reduce the reliability of the app by
+removing resources required for the app to work. Both of which lead to
+negative user experiences.
+
+Thankfully, research by the Chrome team shows that data is very rarely
+cleared automatically by Chrome. It is far more common for users to manually
+clear storage. Thus if a user visits your site regularly, the chances are
+small that your data will be evicted.
+
+To prevent the browser from deleting your data, you can request that your
+entire site's storage be marked persistent. **You should only do this if you
+have critical data that is not backed up to the cloud, for example end-to-end
+encryption keys, that if deleted could result in significant data loss.**
+Persistent storage will not be deleted by the browser, even if storage on the
+device is running tight. It will only be deleted if the user chooses to
+remove it via their site settings.
+
+Persistent storage is [supported in many modern][caniuse-persistent] browsers.
+
+{% Aside %}
+  To learn more about eviction, how much you can store, and how to handle quota
+  limitations, see [Storage for the web](/storage-for-the-web/).
+{% endAside %}
```

Comment:
This is probably less important than the bolded item above. To avoid having two asides close together, add this to one of the other paragraphs.

Reply:
Removed it from the aside.